README.md

```diff
@@ -6,7 +6,7 @@ To make the recipes and layers available, include the following in your `rugpi-b
 
 ```toml
 [repositories]
-tedge-rugpi-core = { git = "https://github.com/thin-edge/tedge-rugpi-core.git" }
+tedge-rugpi-core = { git = "https://github.com/thin-edge/tedge-rugpi-core.git", branch = "v0.7" }
```

Comment:
So the plan is to create a branch "v0.7" is this repository to be used from https://github.com/thin-edge/tedge-rugpi-image?

To be honest, I don't really understand the relationship between these two repositories. And between this PR and that other one: https://github.com/thin-edge/tedge-rugpi-image/pull/79.

Reply:
Thanks. This is helpful. Things are clearer now. And yes it makes sense to use branches for various core images of thin-edge.